kits/arm/ex_teach_repeat.py

```diff
@@ -6,23 +6,37 @@
 from hebi import arm as arm_api
 from hebi.util import create_mobile_io
 
+# Set up to find actuators on the network
+lookup = hebi.Lookup()
+sleep(2)
+
 # Arm setup
 arm_family   = "Arm"
-module_names = ['J1_base', 'J2_shoulder', 'J3_elbow', 'J4_wrist1', 'J5_wrist2', 'J6_wrist3']
-hrdf_file    = "hrdf/A-2085-06.hrdf"
-gains_file   = "gains/A-2085-06.xml"
+module_names = ['J1_base', 'J2A_shoulder1', 'J3_shoulder2', 'J4_elbow1', 'J5_elbow2', 'J6_wrist1', 'J7_wrist2']
+hrdf_file    = "hrdf/A-2303-01.hrdf"
+gains_file   = "gains/A-2303-01.xml"
 
 
 # Create Arm object
 arm = arm_api.create([arm_family],
                      names=module_names,
                      hrdf_file=hrdf_file)
+
+mirror_group = lookup.get_group_from_names([arm_family], ['J2B_shoulder1'])
+while mirror_group is None:
+  print("Looking for double shoulder module...")
+  sleep(1)
+  mirror_group = lookup.get_group_from_names([arm_family], ['J2B_shoulder1'])
+
+# mirror the position/velocity/effort of module 1 ('J2A_shoulder1') to the module
+# in the mirror group ('J2B_shoulder1')
+# Keeps the two modules in the double shoulder bracket in sync
+arm.add_plugin(arm_api.DoubledJointMirror(1, mirror_group))
+
```

Comment:
Do we want this to be the default demo, or a second/optional "7 dof" teach repeat demo?  Maybe parameterized to support both?

Reply:
(same with other examples)